### PR TITLE
Make the installation script idempotent

### PR DESCRIPTION
**Pull request by volAV**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -4,48 +4,69 @@
...
+      git pull
+    fi
 
     # Link .tmux.conf
-    ln -s ${DIR}/tmux/.tmux.conf ${HOME}
+    ln -sf ${DIR}/tmux/.tmux.conf ${HOME}
 }
 
 # Bash
-ln -s ${DIR}/bash/.bashrc ${HOME}
```
Comment (mshkrebtan) [inline install.sh]: @valAV Why do you remove this command?

`.bashrc` is executed when an interactive non-login shell starts.

**Review by mshkrebtan — COMMENTED** [review]
We have to clarify the need of Bash startup files.

File: install.sh (hunk 1 above)
Comment (volAV) [reply install.sh]: macOS terminal by default runs a login-shell and _.bashrc_ will never be used.

**Comment by mshkrebtan** [comment]
Closing this as I am switching to `make` for installation.

**Pull request closed without merging**